src/compiler/transformers/rewrite-aliased-paths.ts

```diff
@@ -155,7 +155,7 @@ function rewriteAliasedImport(
   // from the import path
   const extensionRegex = new RegExp(
     Object.values(ts.Extension)
-      .map((extension) => `${extension}$`)
```

Comment:
Can we add a comment explaining what/why we're doing this? Basically just stealing the PR summary content:
```
// escape bare periods (.) so we don't accidentally use that as a wildcard when matching
```

Reply:
sure, just added a comment along those lines